### PR TITLE
Added few more common usernames in unix_users.txt

### PR DESCRIPTION
**Pull request by adarsh-jha-dev**
I have added some few more common usernames of the Unix users which i often come across and were not listed in the file unix_users.txt .

### CODE REVIEW

File: wordlists/usernames/unix_users.txt [hunk 1]
```diff
@@ -35,7 +36,16 @@ diag
...
 dnsmasq
+david
 dradis
+david
+david
+david
+david
+david
+david
+david
```
Comment (kkrypt0nn) [inline wordlists/usernames/unix_users.txt:47]: It would be awesome if you could clean those duplicates

**Review by kkrypt0nn — CHANGES_REQUESTED** [review]
Looks to me, just those duplicates that could be removed 👍

**Comment by adarsh-jha-dev** [comment]
I have removed the duplicates.

**Comment by kkrypt0nn** [comment]
> I have removed the duplicates.

`dradis` should stay, just the `david` duplicates should be removed 👍

**Comment by adarsh-jha-dev** [comment]
Please let me know if there are any other changes required.

**Comment by kkrypt0nn** [comment]
You've removed some usernames that should stay.
Take the original word list back and add your new usernames again, make sure you *add*, and not *remove* existing usernames.

**Comment by adarsh-jha-dev** [comment]
I have retained the previously present names and added the new ones without duplication.

**Review by kkrypt0nn — CHANGES_REQUESTED** [review]

**Review by kkrypt0nn — APPROVED** [review]
Thanks for your contribution and fixing everything ✨

**Comment by adarsh-jha-dev** [comment]
Thanks for the approval.

**Pull request merged**